### PR TITLE
Updated Issue #716

### LINKED ISSUE #716: Adding an option of copying the referral code beside the user's referral code
**Describe the feature you'd like**

An image button can be added beside referral code which copies the referral code to clipboard when tapped.

**Screenshots**

Screenshot (151)
Just like this, we can change the background of the icon to transparent.

**Additional context**

Please review this and let me know if this is valid feature and can be added to the app.
Many apps having referral code feature have this option of copying to clipboard other than just share button.

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by adiig7**
Fixes #716 

**Changes:**
1. Added an image(ic_content_copy) in drawable folder.
2. Updated the xml file to add that image beside the referral code.
3. Updated the Referral Activity by setting onClick for the image.

Screenshots for the change:
Screenshot_20200516-110346_(Debug)Coding Blocks
Screenshot_20200516-110552_Samsung Notes

### CODE REVIEW

File: app/src/main/java/com/codingblocks/cbonlineapp/course/checkout/CheckoutOrderDetailsFragment.kt [hunk 1]
```diff
@@ -77,6 +77,6 @@ class CheckoutOrderDetailsFragment : BaseCBFragment(), AnkoLogger {
             }
         }
 
-        exploreBtn.setOnClickListener { requireActivity().finish() }
+        exploreBtn.setOnClickListener {startActivity(intentFor<DashboardActivity>()) }
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/course/checkout/CheckoutOrderDetailsFragment.kt:80]: remove this change

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

**Comment by adiig7** [comment]
@aggarwalpulkit596  sir can you please request changes again? I marked it resolved by mistake.

File: app/src/main/java/com/codingblocks/cbonlineapp/profile/ReferralActivity.kt [hunk 2]
```diff
@@ -14,11 +14,22 @@ import kotlinx.coroutines.launch
 import org.jetbrains.anko.share
 
 class ReferralActivity : BaseCBActivity() {
+    private var myClipboard: ClipboardManager? = null
```
Comment (aggarwalpulkit596) [inline app/src/main/java/com/codingblocks/cbonlineapp/profile/ReferralActivity.kt:17]: Use lazy intialization here

**Review by aggarwalpulkit596 — CHANGES_REQUESTED** [review]

**Comment by adiig7** [comment]
@aggarwalpulkit596 I'm closing this pull request and will add a new pull request by tomorrow with all the required changes.

**Pull request closed without merging**